### PR TITLE
Run integration tests in CI

### PR DESCRIPTION
**Pull request by cbenz**
Connected to #64 

Follows #65

### CODE REVIEW

**Comment by cbenz** [comment]
It seems that chrome driver is installed by default in CircleCI containers.

File: circle.yml [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+    - mkdir --parents ~/cache
+  post:
+    # Only downloads if not already present (--no-clobber option)
+    - cd ~/cache && wget --no-clobber http://selenium-release.storage.googleapis.com/3.3/selenium-server-standalone-3.3.0.jar
+    - java -jar ~/cache/selenium-server-standalone-3.3.0.jar:
+        background: true
+test:
+  pre:
+    - npm start:
+        background: true
```
Comment (MattiSG) [inline circle.yml]: I would recommend you wait for the server to start before starting the tests.

Suggested implementation:

```
   # wait for server
    - wget --retry-connrefused --waitretry=1 --output-document=/dev/null http://localhost:$PORT
```

File: circle.yml [hunk 2]
```diff
@@ -1,3 +1,18 @@
+dependencies:
+  cache_directories:
+    - "~/cache"
+  pre:
+    # Only create directory if not already present (--parents option).
+    - mkdir --parents ~/cache
+  post:
+    # Only downloads if not already present (--no-clobber option)
+    - cd ~/cache && wget --no-clobber http://selenium-release.storage.googleapis.com/3.3/selenium-server-standalone-3.3.0.jar
+    - java -jar ~/cache/selenium-server-standalone-3.3.0.jar:
```
Comment (MattiSG) [inline circle.yml]: If you test only with Chrome on CI, you don't need to install the Selenium server. Just start `chromedriver`.

**Comment by cbenz** [comment]
It seems that SauceLabs has only non-free pricing plans. I'm merging this PR to get it done but we may enhance that in the future.

**Comment by MattiSG** [comment]
https://saucelabs.com/open-source

**Pull request merged**